### ISSUE TITLE
The corpus contains more intents and entities

### ISSUE THREAD

**Issue by jojonki** [xliuhw/NLU-Evaluation-Data#2]
This is a really amazing project!

According to your paper, the corpus contains 25716 utterances, annotated for 64 Intents
and 54 Entity Types. However, I found 68 Intents and 56 Entity Types in the corpus.

Please find the attached file which lists up all intents and entities from the corpus. For example, `audio_volume_other` intent does not appear in Table 5.
[intents and entities.txt](https://github.com/xliuhw/NLU-Evaluation-Data/files/2981851/intents.and.entities.txt)

What do the differences between the corpus and the paper mean?

**Comment by xliuhw** [comment]
Thanks for your comments! Apologies for the later response as having not checked the repository for a while.
Yes, you are right! The NLU-Data-Home-Domain-Annotated-All.csv is the full annotated dataset which contains more types of Intents and Entities than what the paper has reported. what the paper has reported was from our cross-validation results which is the sub-set of the whole data because we couldn't use all of them due to the LUIS 10K training utterance limits, and we wanted to use exact same train/test datasets for evaluations on all platforms/services. And we need to report the Intent and Entity number which are corresponding/consistent to the cross-validation results.

And there were some Intents we didn't use them for the evaluations as they have so fewer samples which would make the training dataset unbalance too much. We published them all together to hopefully be useful to others, and they could extend them for their use.

Our script will generate subset of the data from the full set: NLU-Data-Home-Domain-Annotated-All.csv
based on the configuration.

The cross-validation set we used is in the folder CrossValidation.

Hope this will answer your question, but do feel free to ask if you have any further questions.
